libs/mrwebrtc/src/peer_connection.cpp

```diff
@@ -665,29 +665,27 @@ ErrorOr<Transceiver*> PeerConnection::AddTransceiver(
   return transceiver.get();
 }
 
-bool PeerConnection::SetRemoteDescriptionAsync(
-    const char* type,
+Error PeerConnection::SetRemoteDescriptionAsync(
+    mrsSdpMessageType type,
     const char* sdp,
     RemoteDescriptionAppliedCallback callback) noexcept {
   if (!peer_) {
-    return false;
+    return Error(mrsResult::kInvalidOperation);
```

Comment:
nit: should be kInvalidParameter instead?

Reply:
`peer_` is not a parameter to this method. Instead the method was called in an invalid state, so the operation is invalid. No?

Reply:
Right, disregard this.